app/views/grouped_issue_mailer/more_than_10_errors.html.haml

```diff
@@ -0,0 +1,19 @@
+%p This is a email notifying you that 10 errors occurred in last hour.
+%table#t01{:style => "width: 100%;border: 1px solid black;border-collapse: collapse;"}
+  %tr
+    %th{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;background-color: black;color: white;"} ID
+    %th{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;background-color: black;color: white;"} Platform
+    %th{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;background-color: black;color: white;"} Message
+    %th{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;background-color: black;color: white;"} Created At
+    %th{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;background-color: black;color: white;"} Website
+    %th{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;background-color: black;color: white;"} Event id
+    %th{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;background-color: black;color: white;"} Issue Url
+  - @last_hour_errors.each_with_index do |issue, index|
+    %tr
+      %td{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;"} #{issue.id}
+      %td{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;"} #{issue.platform}
+      %td{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;"} #{issue.message}
+      %td{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;"} #{issue.created_at}
+      %td{:style => "border: 1px solid black;border-collapse: collapse;padding: 5px;text-align: left;"} #{issue.website.domain}
```

Comment:
I added them this way because I remember at SEO Tripwire we had a problem when adding classes to html tags, and this approach solved our issue